Cargo.toml

```diff
@@ -24,15 +24,19 @@ rustc-args = ["--cfg", "docsrs"]
 rustdoc-args = ["--cfg", "docsrs"]
 
 [features]
-default = ["file-drop", "objc-exception", "protocol"]
+default = ["file-drop", "objc-exception", "protocol", "os-webview"]
 objc-exception = ["objc/exception"]
 file-drop = []
 protocol = []
 devtools = []
 transparent = []
 fullscreen = []
-linux-body = ["webkit2gtk/v2_40"]
+linux-body = ["webkit2gtk/v2_40", "os-webview"]
```

Comment:
maybe rename this one to webkit2gtk-body now that we are preparing for other implementations?

if we rename it here, we need to rename in code too